### PR TITLE
Implemented Toss Feature

### LINKED ISSUE #11: Set up team method: Toss

### PR DESCRIPTION
**Pull request by ghost**
The toss Pokemon feature has been added! This feature allows the user to toss a Pokemon from their team and will inform them of what Pokemon has been tossed with caught exceptions for when the user has pressed an invalid key.  Please review to test functionality and ensure that no errors occur.

### CODE REVIEW

File: game/player.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ class Player:
     def __init__(self):
         """Fills the player's team with a random Pokemon from the PokeAPI"""
         if len(self.team) < 1:
-            self.team.extend(asyncio.run(pokeapi.get_random_pokemon_from_api()))
+            self.team.extend(asyncio.run(pokeapi.get_random_pokemon_from_api(6)))
```
Comment (biggestcookie) [inline game/player.py]: This shouldn't be here

Comment (ghost) [reply game/player.py]: Yes it should

Comment (ghost) [reply game/player.py]: Changed to  self.team.extend(asyncio.run(pokeapi.get_random_pokemon_from_api(1)))

Comment (ghost) [reply game/player.py]: Changed to  self.team.extend(asyncio.run(pokeapi.get_random_pokemon_from_api(1))) to remove redundancy

**Comment by biggestcookie** [comment]
Looks good to me

**Pull request merged**